src/frontend/src/handler/cancel_job.rs

```diff
@@ -36,16 +34,14 @@ pub(super) async fn handle_cancel(
         .await?;
     let rows = canceled_jobs
         .into_iter()
-        .map(|id| Row::new(vec![Some(id.to_string().into())]))
-        .collect_vec();
+        .map(|id| CancelRow { id: id as i32 });
     Ok(PgResponse::builder(StatementType::CANCEL_COMMAND)
-        .values(
-            rows.into(),
-            vec![PgFieldDescriptor::new(
-                "Id".to_string(),
-                DataType::Varchar.to_oid(),
-                DataType::Varchar.type_len(),
-            )],
-        )
+        .rows(rows)
         .into())
 }
+
+#[derive(Fields)]
+#[fields(style = "Title Case")]
+struct CancelRow {
+    id: i32,
```

Comment:
Previously it's `Varchar`. Is the change okay?

Reply:
Not sure. cc @yezizp2012

Reply:
I think it's okay to change it.

Reply:
I'm afraid there will be a problem. Values seems to be encoded in text format in simple query, but in binary format in extended mode.

Proof: When I changed a field from varchar to boolean, the values were interpreted incorrectly in extended query. 🤯
https://buildkite.com/risingwavelabs/pull-request/builds/42258#018dc163-569b-4202-96b6-29642970416e/10435-10522

There is no problem with varchar because both text and binary encoding are the same.

Reply:
We didn't consider the encoding format (always assume text format) when handling statement before. 🙃